index.html

```diff
@@ -2133,8 +1145,8 @@ <h2>Resource Integrity</h2>
 property named <dfn class="lint-ignore">`digestMultibase`</dfn> in any object
 that includes an `id` property. If present, the `digestMultibase` value MUST be
 a single string value, or an array of string values that are
-<a href="#multibase-0">Multibase</a>-encoded <a href="#multihash">Multihash</a>
-values.
+<a data-cite="CONTROLLER-DOCUMENT#multibase">Multibase</a>-encoded
+<a data-cite="CONTROLLER-DOCUMENT#multihash">Multihash</a> values.
```

Comment:
travels with the change above
```suggestion
<a data-cite="CONTROLLER-DOCUMENT#multihash">Multihash</a> value.
```